tck/src/test/java/TestTokenWithGroupsPath.java

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
I think there are quite a few duplicate licenses in the files.

Can you check them?

Reply:
Sure will do

Reply:
@kenfinnigan looks like the default package was causing it so I moved those test classes into a non empty package; also updated all the license headers to have 2019 and starting with a single asterisk